packages/http-client-csharp/emitter/src/constants.ts

```diff
@@ -1,9 +1,26 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+import { CreateSdkContextOptions } from "@azure-tools/typespec-client-generator-core";
+
 export const projectedNameJsonKey = "json";
 export const projectedNameCSharpKey = "csharp";
 export const projectedNameClientKey = "client";
 export const mockApiVersion = "0000-00-00";
 export const tspOutputFileName = "tspCodeModel.json";
 export const configurationFileName = "Configuration.json";
+export let createSDKContextoptions: CreateSdkContextOptions = {};
```

Comment:
As we leverage TCGC types and functions, can we use the similar pattern for those names in dotnet emitter?
e.g.   createSdkContextOptions,   setSdkContextOptions.